source/blog/2017-08-21-ember-2-15-released.md

```diff
@@ -0,0 +1,392 @@
+
+---
+
+## Ember Data
+
+Ember Data is the official data persistence library for Ember.js applications.
+
+### Changes in Ember Data 2.15
+
+Ember Data 2.14 contained a number of performance improvements summarized
+in the [2.14 release
+post](https://emberjs.com/blog/2017/07/06/ember-2-14-released.html#toc_changes-in-ember-data-2-14).
+Most of the development effort in the beta cycle has been to address regressions
+introduced by those changes. 2.15.0 contains no new features or deprecations,
+but does include the same bugfixes as can be found in 2.14.10.
+
+For details on changes in Ember Data 2.15.0, please review the
+[Ember Data 2.15.0 release page](https://github.com/emberjs/data/releases/tag/v2.15.0).
+
+For details on the upcoming changes in Ember Data 2.16, please review the
+[Ember Data 2.16.0-beta.1 release page](https://github.com/emberjs/data/releases/tag/v2.16.0-beta.1).
+
+---
+
+## Ember CLI
+
+Ember CLI is the command line interface for managing and packaging Ember.js
+applications.
+
+### Upgrading Ember CLI
+
+You may upgrade Ember CLI separately from Ember.js and Ember Data!
+There is a new experimental tool for Ember CLI upgrades called
+[ember-cli-update](https://github.com/kellyselden/ember-cli-update.git).
+To use it, run this command to install it globally:
+
+```
+npm install -g ember-cli-update
+```
+
+Then run:
+
+```
+ember-cli-update
+```
+
+It runs your system git merge tool if it finds a conflict. This can be pretty
+overwhelming for beginners, so you can run
+
+```
+ember-cli-update --ignore-conflicts
+```
+
+to handle the conflicts yourself.
+
+If this new tool is giving you problems, you can still upgrade your projects
+manually. To upgrade your projects using `yarn` run:
+
+```
+yarn upgrade ember-cli
+```
+
+To upgrade your projects using `npm` run:
+
+```
+npm install --save-dev ember-cli
+```
+
+After running the
+upgrade command run `ember init` inside of the project directory to apply the
+blueprint changes. You can preview those changes for [applications](https://github.com/ember-cli/ember-new-output/compare/v2.14.0...v2.15.0)
+and [addons](https://github.com/ember-cli/ember-addon-output/compare/v2.14.0...v2.15.0).
+
+### Changes in Ember CLI 2.15
+
+#### Chrome by default
+
+Ember CLI will configure new apps to run tests in [headless Chrome](https://chromium.googlesource.com/chromium/src/+/lkgr/headless/README.md)
+by default, instead of PhantomJS.
+
+PhantomJS has served the wider JavaScript community well for a long time, being
+a practical alternative to running browsers headless via tools like Xvbf. It
+has, however, been a proxy for what we really want to test– the browsers that
+users are running.
+
+Now that we can easily test in headless Chrome the motivation for using
+PhantomJS has diminished, and as a result it is [no longer actively maintained](https://chromium.googlesource.com/chromium/src/+/lkgr/headless/README.md).
```

Comment:
I don't think this link should be the same as the one above.
My guess is it should be pointing to something about phantomjs?